themes/default/content/blog/tf-migration-offer/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Terraform Migration Offer"
+date: 2023-06-12T20:05:11Z
+meta_desc: The Terraform Migration Offer provides Terraform workspace migrations as part of Pulumi Enterprise and Business Critical onboarding. 
+meta_image: meta.png
+authors:
+    - aaron-kao
+tags:
+    - terraform
+    - migration
+---
+
+Many of our users come to Pulumi from existing infrastructure as code (IaC) tools like Terraform. They choose Pulumi because of the increased developer productivity and ability to harness the complexity of the cloud when using real programming languages. However, migrating existing IaC projects to Pulumi can come with a lot of challenges and time. Today, we announced converting full Terraform projects to Pulumi directly within the Pulumi CLI. Previously, the `tf2pulumi` tool could convert just snippets of Terraform to Pulumi. With the release, you can now convert the vast majority of complete Terraform projects. See Justin Van Patten’s [blog post](/blog/converting-full-terraform-programs-to-pulumi) for more details.
```

Comment:
```suggestion
Many of our users come to Pulumi from existing infrastructure as code (IaC) tools like Terraform. They choose Pulumi because of the increased productivity and ability to tame the complexity of the cloud thanks to the choice of any programming language. However, migrating existing IaC projects to Pulumi can come with challenges and time. Today, [we announced](/blog/converting-full-terraform-programs-to-pulumi) the ability to convert entire Terraform projects and modules to Pulumi directly from the Pulumi CLI. This can make migrations more seamless. That said, many enterprises we work with have a lot of code to convert and not much time in which to do it. Today we are excited to announce a new Terraform Migration Offer, which includes migration services from our expert customer engineers, bundled as part of Pulumi Enterprise and Business Critical Editions.
```

Remember, the opening paragraph becomes the preview text - we want to include the "news" in it. I realize this means the next paragraph has to change a bit.